File: server_spring/src/test/java/de/werkelmann/rekote/server/CustomTrayIconTest.java
package de.werkelmann.rekote.server;

import org.junit.Test;

import java.awt.Image;

import static org.junit.Assert.*;

public class CustomTrayIconTest {

    @Test
    public void testIfDefaultIconCouldBeLoaded() {
        Image i = CustomTrayIcon.getDefaultImage();
        assertNotNull(i);
    }
}

File: server_spring/src/test/kotlin/de/werkelmann/rekote/server/CustomTrayIconTest.kt
package de.werkelmann.rekote.server

import org.junit.Assert.assertNotNull
import org.junit.Test

class CustomTrayIconTest {

    @Test
    fun testIfDefaultIconCouldBeLoaded() {
        val i = CustomTrayIcon.defaultImage
        assertNotNull(i)
    }
}